src/osmf/lib/HDSPlayer.as

```diff
@@ -430,6 +432,7 @@ package
       if (_playerState == MediaPlayerState.READY || _playerState == MediaPlayerState.PAUSED 
           || _playerState == MediaPlayerState.BUFFERING)
       {
+        _playCompleted = false;
```

Comment:
this looks good but i wonder also if this may be good to set to false when the video url changes but before play is called